### PR TITLE
Edit MistServer license and description

### PR DESCRIPTION
**Pull request by stronk-dev**
Hi, I am one of the maintainers for MistServer. Just wanted to edit the details of our software, as the software has been fully open sourced and un-licensed. Awesome for self hosters who need an enterprise-ready media engine :)

### CODE REVIEW

**Review by kokomo123 — APPROVED** [review]
No issues here. Thanks for the edit and thanks for your contribution!

File: software/mistserver.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name: MistServer
-website_url: https://mistserver.org/
-description: Streaming media server that works well in any streaming environment.
+website_url: https://mistserver.com/
```
Comment (nodiscc) [inline software/mistserver.yml:2]: I think the original `.org` domain is actually a better resource for self-hosting than the .com website which does not have any kind of technical documentation/info.

Comment (kokomo123) [reply software/mistserver.yml:2]: this makes sense tbh

Comment (Ki-er) [reply software/mistserver.yml:2]: Still includes a description of what the app is. Id also stick with the `.org`

Comment (nodiscc) [inline software/mistserver.yml:2]:
```suggestion
website_url: https://mistserver.org/
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/mistserver.yml (hunk 1 above)
Comment (stronk-dev) [reply software/mistserver.yml]: Yeah that makes sense, we'll make sure to give the `.org` website a visual touch up as well sometime soon but at least it links to all developer resources

**Review by kokomo123 — APPROVED** [review]
LGTM, no issues here! Thank you for your contribution 👍🏻

**Review by nodiscc — APPROVED** [review]

**Pull request merged**